src/obis-observer/commands/downlink/GetMeterArchiveProfile.ts

```diff
@@ -0,0 +1,81 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import {REQUEST_ID_SIZE, ICommandParameters} from '../../CommandBinaryBuffer.js';
+import {DOWNLINK} from '../../constants/directions.js';
+
+
+/**
+ * IGetMeterArchiveProfileParameters command parameters
+ */
+interface IGetMeterArchiveProfileParameters extends ICommandParameters {
+    meterProfileId: number
+}
+
+
+const COMMAND_ID = 0x66;
+const COMMAND_SIZE = REQUEST_ID_SIZE + 1;
+
+const examples: TCommandExampleList = [
+    {
+        name: 'get meter archive profile',
+        parameters: {
+            requestId: 3,
+            meterProfileId: 4
```

Comment:
```suggestion
            meterId: 4
```

?

or it will return in response all archive profile from meters with these profileId?

Reply:
it will return the archive settings from this meter profile